spawn_test.go

```diff
@@ -1,14 +1,14 @@
 package main
 
 import (
-	"strings"
 	"testing"
+	"regexp"
 
 	"github.com/miquella/vaulted/lib"
 )
 
 func TestSpawn(t *testing.T) {
-	spawnRefreshVar := `GOPATH="/vaulted"`
+	spawnRefreshVar := `GOPATH=["|']/vaulted`
```

Comment:
Isn't the `|` wrong? `["']` should be correct? Otherwise you will also match `|/valued` which I don't think you want to do.

Reply:
indeed, fixed!